skglm/estimators.py

```diff
@@ -837,19 +840,189 @@ class MCPRegression(LinearModel, RegressorMixin):
     """
 
     def __init__(self, alpha=1., gamma=3, max_iter=50, max_epochs=50_000, p0=10,
-                 verbose=0, tol=1e-4, fit_intercept=True, warm_start=False,
-                 ws_strategy="subdiff"):
+                 verbose=0, tol=1e-4, positive=False, fit_intercept=True,
+                 warm_start=False, ws_strategy="subdiff"):
         super().__init__()
         self.alpha = alpha
         self.gamma = gamma
-        self.tol = tol
         self.max_iter = max_iter
         self.max_epochs = max_epochs
         self.p0 = p0
-        self.ws_strategy = ws_strategy
+        self.verbose = verbose
+        self.tol = tol
+        self.positive = positive
         self.fit_intercept = fit_intercept
         self.warm_start = warm_start
+        self.ws_strategy = ws_strategy
+
+    def path(self, X, y, alphas, coef_init=None, return_n_iter=True, **params):
```

Comment:
do you need it enough that the code addition is justified ? i.e. calling fit repeatedly with warm start is not an option  ?

Reply:
Since `WeightedMCPRegression` and `MCPRegression` were merged, I guess this comment is fixed